src/components/DropDown.tsx

```diff
@@ -48,7 +48,7 @@ class DropDown extends React.PureComponent<DropdownProps, DropdownState> {
             labelComponent({ isOpen, toggleDropdown: this.toggleDropdown })
           ) : (
             <DropDownButton
-              isSelected={isSelected}
+              isSelected={!!isSelected}
```

Comment:
why is this needed? isn't isSelected always boolean ?

Reply:
No its optional in DropdownProps, so can be undefined. 😢

Reply:
ok